src/scripts/index.js

```diff
@@ -5,43 +5,36 @@ import ReactDOM from 'react-dom'
 import { AppContainer } from 'react-hot-loader'
 import { Provider } from 'react-redux'
 
+import { initParatiiLib } from 'utils/ParatiiLib'
 import Root from 'components/Root'
+import { getRoot } from 'utils/AppUtils'
 import createStore from 'scripts/createStore'
 import 'styles/app.scss'
 
-console.log(process.env.NODE_ENV)
-
-let root: ?Element = document.getElementById('root')
-
 const store = createStore()
 
-if (!root) {
-  root = document.createElement('div')
-  root.setAttribute('id', 'root')
-  document.body && document.body.appendChild(root)
-}
-
-ReactDOM.render(
-  <Provider store={store}>
-    <AppContainer>
-      <Root />
-    </AppContainer>
-  </Provider>,
-  root
-)
+initParatiiLib()
```

Comment:
Here is an example of initializing paratii-lib before rendering the app

Reply:
And I may want to think of another approach to this, as this noticeably slows down the rendering on the FE.